### DISCUSSION TITLE
I want to operate"division" between two shapes, & the result to have to have one path in color and one path transparent?

### DISCUSSION THREAD

**Discussion by MarySofy** [spakin/SimpInkScr#104] [Q&A] [answered]
In the code below, have two circles that operate 'division', but I wonder if it possible to assign colors to the result? I would like one path to be a color and the other path to be transparent / no-fill. 

CODE )

c1 = circle((60, 60), 50, fill='magenta', stroke_width=4).to_path()
c2 = circle((90, 60), 50, fill='cyan', stroke_width=4).to_path()
objs = apply_path_operation('division', [c1, c2])

I was recommended to try python zip function which would assign different colors. But it doesn't work.
color_list = ['red', 'green', 'blue']
objs = apply_path_operation('division', [c1, c2])
for o, color in zip(objs, color_list):
o.style(fill=color)

**Comment by spakin** [comment, marked as answer]
`zip` is a perfectly reasonable approach.  To produce a transparent object you can specify its color as `none`, as in the following code:
```Python
c1 = circle((60, 60), 50, fill='magenta', stroke_width=4).to_path()
c2 = circle((90, 60), 50, fill='cyan', stroke_width=4).to_path()
color_list = ['red', 'none']
objs = apply_path_operation('division', [c1, c2])
for o, color in zip(objs, color_list):
    o.style(fill=color)
```
Does that do what you want?

**Reply by MarySofy** [reply]
Thanks alot! it does work!
